# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bluez-libs/glibc-2.9.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/sdp.c.orig	2009-10-06 13:39:56.000000000 +0200
+++ ./src/sdp.c	2009-10-06 13:40:22.000000000 +0200
@@ -35,6 +35,7 @@
 #include <stdlib.h>
 #include <malloc.h>
 #include <syslog.h>
+#include <limits.h>
 #include <sys/un.h>
 #include <sys/socket.h>
 #include <sys/time.h>
